src/Illuminate/Foundation/Vite.php

```diff
@@ -397,4 +419,53 @@ public function reactRefresh()
             )
         );
     }
+
+    /**
+     * Merge the given entry points.
+     *
+     * @param  array  $entryPoints
+     * @return $this
+     */
+    public function withEntryPoints(array $entryPoints)
+    {
+        $this->entryPoints = array_merge($this->entryPoints, $entryPoints);
+
+        return $this;
+    }
+
+    /**
+     * Set the hot file path attribute.
+     *
+     * @param  string
+     * @return $this
+     */
+    public function useHotFile(string $path)
+    {
+        $this->hotFilePath = Str::start($path, '/');
+
+        return $this;
+    }
+
+    /**
+     * Set the build directory attribute.
+     *
+     * @param  string
+     * @return $this
+     */
+    public function useBuildDirectory(string $path)
+    {
+        $this->buildDirectory = Str::start($path, '/');
```

Comment:
Same here re: not prefixing with a slash.

Reply:
Well, then we should remove it from the `__invoke` as well, I guess.